tools/scripts/docs-website/git-commands.js

```diff
@@ -76,7 +76,7 @@ async function syncWithParent () {
 }
 
 async function pushToRemote (remote, branchName) {
```

Comment:
I dont think the remote param is needed any more, right?

Reply:
Yes can be removed